src/main/java/me/zeroeightsix/fiber/builder/ConfigValueBuilder.java

```diff
@@ -69,16 +105,39 @@ public static boolean isAggregate(Class<?> type) {
         this.type = type;
     }
 
+    /**
+     * Sets the name.
+     * <br> If {@code null}, or if this method is never called, the {@code ConfigValue} won't have a name. Thus, it might be ignored during (de)serialisation. It also won't be able to be found by name in its parent node.
+     *
+     * @param name the name
+     * @return {@code this} builder
+     * @see Node#lookup
+     */
     public B withName(String name) {
         this.name = name;
         return self();
     }
 
+    /**
+     * Sets the comment.
```

Comment:
Same as above. Even though this is in a class and as such it may be relatively understandable I'd say adding what this set's a comment for is useful.